openml/tasks/task.py

```diff
@@ -96,6 +96,28 @@ def get_split_dimensions(self):
 
         return self.split.repeats, self.split.folds, self.split.samples
 
+    def push_tag(self, tag):
+        """Annotates this flow with a tag on the server.
+
+        Parameters
+        ----------
+        tag : str
+            Tag to attach to the flow.
+        """
+        data = {'flow_id': self.flow_id, 'tag': tag}
+        _perform_api_call("/flow/tag", data=data)
+
+    def remove_tag(self, tag):
+        """Removes a tag from this flow on the server.
+
+        Parameters
+        ----------
+        tag : str
+            Tag to attach to the flow.
+        """
+        data = {'flow_id': self.flow_id, 'tag': tag}
+        _perform_api_call("/flow/untag", data=data)
```

Comment:
And here.

Reply:
looks like I missed one search and replace.